src/components/media-views.js

```diff
@@ -499,13 +493,12 @@ AFRAME.registerComponent("media-video", {
         if (texture.hls.currentLevel >= 0) {
           updateLiveState();
         }
+      } else {
+        this.videoIsLive = this.video.duration === Infinity;
```

Comment:
No clue if this is actually going to work, but I can't get a streaming source to work locally to try it out.